cmd/kubeconfigwriter/main.go

```diff
@@ -32,7 +32,8 @@ import (
 )
 
 var (
-	clusterConfig = flag.String("clusterConfig", "", "json string with the configuration of a cluster based on values from a cluster resource. Only required for external clusters.")
+	clusterConfig  = flag.String("clusterConfig", "", "json string with the configuration of a cluster based on values from a cluster resource. Only required for external clusters.")
+	destinationDir = flag.String("destinationDir", "", "destination directory where generated kubeconfig file will be stored.")
```

Comment:
Where does this new parameter get used?

Reply:
`destinationDir` will be used in the task, which will replace PipelineResourceType `Cluster` to a `cluster-task`. 
To optimize the number of  steps in the task, I am using this new parameter. 
ref - https://github.com/tektoncd/catalog/pull/251